### PR TITLE
Add omUSD token metadata

### PR DESCRIPTION
**Pull request by Hirama**
# Add omUSD token metadata

ERC-20: https://etherscan.io/address/0xd3043D66Afe00344C115f7f81D18277C5c718FF8
NEP-141: https://nearblocks.io/address/d3043d66afe00344c115f7f81d18277c5c718ff8.factory.bridge.near
Aurora ERC-20 (BlockScout): https://explorer.aurora.dev/address/0xFFc4c2429B6fF6D748523E95cdaf8D9d82fDE7AA
Aurora ERC-20 (Aurorascan): https://aurorascan.dev/address/0xFFc4c2429B6fF6D748523E95cdaf8D9d82fDE7AA

### CODE REVIEW

**Comment by Karkunow** [comment]
@Hirama Hey! We merge PRs every 2-3 weeks or so. I have let the devs know. They will answer you on GitHub

**Comment by Karkunow** [comment]
We have received a ticket from Andrei ([az@openmoneyado.xyz]). But I can't reach him via this email.

File: tokens/omusd.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
...130 5 193 27 100 36 173 82 246 154 330 331 199 886 -246 1041 -92 32 -241 40 -337 18z m79 -114 c0 -27 -4 -31 -56 -46 -108 -31 -173 -95 -194 -191 -19 -83 11 -158 88 -224 l49 -42 -53 -50 c-67 -63 -105 -138 -70 -138 8 0 17 11 20 25 7 28 64 95 107 126 l28 20 51 -29 50 -29 -55 -38 c-31 -21 -60 -46 -66 -57 -15 -27 7 -23 50 10 20 16 53 39 74 51 l37 23 48 -31 c72 -46 104 -83 100 -117 -7 -59 -113 -98 -206 -74 -48 12 -92 43 -92 67 0 10 -21 13 -84 13 l-84 0 9 -41 c14 -60 33 -93 80 -132 40 -35 113 -67 151 -67 13 0 18 -8 18 -30 0 -28 -2 -29 -32 -23 -111 20 -207 71 -287 153 -203 206 -202 538 3 743 80 80 208 145 309 156 4 1 7 -12 7 -28z m225 2 c145 -44 274 -170 335 -328 20 -50 24 -78 24 -175 0 -97 -4 -125 -24 -175 -61 -158 -190 -284 -335 -328 -101 -31 -105 -31 -105 0 0 26 5 29 50 40 65 16 126 57 164 111 25 35 31 55 34 111 5 84 -15 129 -87 193 l-50 45 40 32 c63 51 120 151 86 151 -8 0 -20 -15 -26 -32 -12 -35 -105 -128 -127 -128 -16 0 -104 42 -104 49 0 4 19 18 43 33 23 15 53 39 67 53 39 41 -1 32 -62 -15 -29 -22 -63 -43 -75 -46 -27 -7 -102 35 -142 82 -27 30 -29 37 -19 65 30 87 252 93 290 8 13 -28 14 -29 90 -29 77 0 78 0 78 25 0 90 -90 183 -208 214 -37 11 -42 15 -42 42 0 33 3 33 105 2z'/%3E%3C/svg%3E",
```
Comment (karim-en) [inline tokens/omusd.json:8]:
```suggestion
    "icon": "data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0idXRmLTgiPz4KPHN2ZyB2ZXJzaW9uPSIxLjEiIGlkPSJMYXllcl8xIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciCiAgICB4bWxuczp4bGluaz0iaHR0cDovL3d3dy53My5vcmcvMTk5OS94bGluayIgeD0iMHB4IiB5PSIwcHgiIHZpZXdCb3g9IjAgMCAyNTYgMjU2IgogICAgc3R5bGU9ImVuYWJsZS1iYWNrZ3JvdW5kOm5ldyAwIDAgMjU2IDI1NjsiIHhtbDpzcGFjZT0icHJlc2VydmUiPgogICAgPGcgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMC4wMDAwMDAsMjU2LjAwMDAwMCkgc2NhbGUoMC4xMDAwMDAsLTAuMTAwMDAwKSIgZmlsbD0iIzAwRkYwMCIgc3Ryb2tlPSJub25lIj4KICAgICAgICA8cGF0aCBkPSJNODYxIDE2MTUgYy00MTQgLTk0IC02MTggLTU2NyAtNDA0IC05MzUgNzYgLTEyOSAyMTIgLTIzNiAzNTggLTI4MwogICAgICAgIDU3IC0xOCA5MyAtMjIgMTkwIC0yMiAxMDQgMSAxMzAgNSAxOTMgMjcgMTAwIDM2IDE3MyA4MiAyNDYgMTU0IDMzMCAzMzEgMTk5CiAgICAgICAgODg2IC0yNDYgMTA0MSAtOTIgMzIgLTI0MSA0MCAtMzM3IDE4eiBtNzkgLTExNCBjMCAtMjcgLTQgLTMxIC01NiAtNDYgLTEwOAogICAgICAgIC0zMSAtMTczIC05NSAtMTk0IC0xOTEgLTE5IC04MyAxMSAtMTU4IDg4IC0yMjQgbDQ5IC00MiAtNTMgLTUwIGMtNjcgLTYzCiAgICAgICAgLTEwNSAtMTM4IC03MCAtMTM4IDggMCAxNyAxMSAyMCAyNSA3IDI4IDY0IDk1IDEwNyAxMjYgbDI4IDIwIDUxIC0yOSA1MCAtMjkKICAgICAgICAtNTUgLTM4IGMtMzEgLTIxIC02MCAtNDYgLTY2IC01NyAtMTUgLTI3IDcgLTIzIDUwIDEwIDIwIDE2IDUzIDM5IDc0IDUxIGwzNwogICAgICAgIDIzIDQ4IC0zMSBjNzIgLTQ2IDEwNCAtODMgMTAwIC0xMTcgLTcgLTU5IC0xMTMgLTk4IC0yMDYgLTc0IC00OCAxMiAtOTIgNDMKICAgICAgICAtOTIgNjcgMCAxMCAtMjEgMTMgLTg0IDEzIGwtODQgMCA5IC00MSBjMTQgLTYwIDMzIC05MyA4MCAtMTMyIDQwIC0zNSAxMTMKICAgICAgICAtNjcgMTUxIC02NyAxMyAwIDE4IC04IDE4IC0zMCAwIC0yOCAtMiAtMjkgLTMyIC0yMyAtMTExIDIwIC0yMDcgNzEgLTI4NyAxNTMKICAgICAgICAtMjAzIDIwNiAtMjAyIDUzOCAzIDc0MyA4MCA4MCAyMDggMTQ1IDMwOSAxNTYgNCAxIDcgLTEyIDcgLTI4eiBtMjI1IDIgYzE0NQogICAgICAgIC00NCAyNzQgLTE3MCAzMzUgLTMyOCAyMCAtNTAgMjQgLTc4IDI0IC0xNzUgMCAtOTcgLTQgLTEyNSAtMjQgLTE3NSAtNjEgLTE1OAogICAgICAgIC0xOTAgLTI4NCAtMzM1IC0zMjggLTEwMSAtMzEgLTEwNSAtMzEgLTEwNSAwIDAgMjYgNSAyOSA1MCA0MCA2NSAxNiAxMjYgNTcKICAgICAgICAxNjQgMTExIDI1IDM1IDMxIDU1IDM0IDExMSA1IDg0IC0xNSAxMjkgLTg3IDE5MyBsLTUwIDQ1IDQwIDMyIGM2MyA1MSAxMjAKICAgICAgICAxNTEgODYgMTUxIC04IDAgLTIwIC0xNSAtMjYgLTMyIC0xMiAtMzUgLTEwNSAtMTI4IC0xMjcgLTEyOCAtMTYgMCAtMTA0IDQyCiAgICAgICAgLTEwNCA0OSAwIDQgMTkgMTggNDMgMzMgMjMgMTUgNTMgMzkgNjcgNTMgMzkgNDEgLTEgMzIgLTYyIC0xNSAtMjkgLTIyIC02MwogICAgICAgIC00MyAtNzUgLTQ2IC0yNyAtNyAtMTAyIDM1IC0xNDIgODIgLTI3IDMwIC0yOSAzNyAtMTkgNjUgMzAgODcgMjUyIDkzIDI5MCA4CiAgICAgICAgMTMgLTI4IDE0IC0yOSA5MCAtMjkgNzcgMCA3OCAwIDc4IDI1IDAgOTAgLTkwIDE4MyAtMjA4IDIxNCAtMzcgMTEgLTQyIDE1CiAgICAgICAgLTQyIDQyIDAgMzMgMyAzMyAxMDUgMnoiLz4KICAgIDwvZz4KPC9zdmc+",
```

**Review by karim-en — CHANGES_REQUESTED** [review]

File: tokens/omusd.json (hunk 1 above)
Comment (karim-en) [reply tokens/omusd.json:8]: let's use the base64 format

**Comment by Hirama** [comment]
@karim-en done

**Review by karim-en — APPROVED** [review]

**Pull request merged**